docs/source/usage.rst

```diff
@@ -371,3 +371,82 @@ without the parameter part, for example:
 Note that using the fully qualified test name, which would include the
 parameter (in this case ``test_second[1]``, ``test_second[2]`` etc) is not
 supported.
+
+
+Multiple test order markers
+-------------------
+More than one order marker can be set for a test.
+In this scenario test will be executed several times in defined order.
+
+Combination of absolute and relative ordering
+~~~~~~~~~~~~~~
+.. code:: python
+
+ import pytest
+
+ @pytest.mark.order(1)
+ @pytest.mark.order(-1)
+ def test_one_and_seven():
+     pass
+
+ @pytest.mark.order(2)
+ @pytest.mark.order(-2)
+ def test_two_and_six():
+     pass
+
+ def test_four():
+     pass
+
+ @pytest.mark.order(before="test_four")
+ @pytest.mark.order(after="test_four")
+ def test_three_and_five():
+     pass
+
+Each order marker is converted to a parameter set:
+
+::
+
+    ============================= test session starts =============================
+    collecting ... collected 7 items
+    test_multiple_markers.py::test_one_and_seven[index=1]
+    test_multiple_markers.py::test_two_and_six[index=2]
+    test_multiple_markers.py::test_three_and_five[before=test_four]
+    test_multiple_markers.py::test_four
+    test_multiple_markers.py::test_three_and_five[after=test_four]
+    test_multiple_markers.py::test_two_and_six[index=-2]
+    test_multiple_markers.py::test_one_and_seven[index=-1]
+    ============================== 7 passed in 0.02s ==============================
+
+
+Parametrized tests
+~~~~~~~~~~~~~~
+Although multiple test order markers creates its own parametrization, it can be used with parametrized tests.
```

Comment:
creates its own -> create their own

Good point by the way to include this here - I also had been thinking about possible problems there.